hack/update-codegen.sh

```diff
@@ -43,5 +43,16 @@ done
 echo "Generating deepcopy funcs"
 ${BIN_DIR}/deepcopy-gen --input-dirs "$(codegen::join , "${FQ_APIS[@]}")" -O zz_generated.deepcopy --bounding-dirs "${APIS_PKG}" -h $BOILERPLATE
 
-echo "Generating CRDs in ./deploy/charts/trust-manager/templates"
-${BIN_DIR}/controller-gen crd schemapatch:manifests=./deploy/charts/trust-manager/templates output:dir=./deploy/charts/trust-manager/templates paths=./pkg/apis/...
+echo "Generating CRDs in ./deploy/crds"
+${BIN_DIR}/controller-gen crd schemapatch:manifests=./deploy/crds output:dir=./deploy/crds paths=./pkg/apis/...
+
+echo "Updating CRDs with helm templating, writing to /deploy/charts/trust-manager/templates"
```

Comment:
nitpick: The path looks like it's wrong maybe?

```suggestion
echo "Updating CRDs with helm templating, writing to ./deploy/charts/trust-manager/templates"
```